### PR TITLE
[WIP][12.0][ADD] web_pwa_cache

### PR DESCRIPTION
**Pull request by Tardo**
Enables support for configuring PWA offline/cached mode

- Adds a new form type called 'formPWA' that wins to "form" when user runs in standalone mode.
- Manage cache
- Support offline operations

cc @tecnativa TT23822

### CODE REVIEW

File: web_pwa_cache/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
+=======================
```
Comment (pedrobaeza) [inline web_pwa_cache/README.rst]: This readme is totally incorrect

**Comment by rousseldenis** [comment]
@Tardo Any news on this ?

**Comment by pedrobaeza** [comment]
We are still working on it. We will have news soon.

**Comment by rousseldenis** [comment]
> We are still working on it. We will have news soon.

Ok, I was reviewing other PR's that depend on this.

**Comment by pedrobaeza** [comment]
Yes, we have published them for checking on a real environment, but they are subjected to change. The rest of the modules are basically "data" to feed this one.

**Comment by pedrobaeza** [comment]
Travis is red

**Comment by sergiocorato** [comment]
@Tardo thanks for the contribution!
I was trying the PR, how can I activate offline mode?

**Comment by pedrobaeza** [comment]
Right now it's disabled, as this is still WIP and we are improving the performance changing the core storage. We'll remove the WIP when done.

**Comment by sergiocorato** [comment]
I'll try it later then, thanks.

**Comment by Tardo** [comment]
Closed. Continue in #1863

**Pull request closed without merging**